tilewe/tournament.py

```diff
@@ -418,14 +418,14 @@ def _play_game(self, player_to_engine: list[int]) -> tuple[list[int], list[int],
         try: 
             engine_to_player = { value: key for key, value in enumerate(player_to_engine) }
             while not board.finished: 
-                engine = self.engines[player_to_engine[board.current_player]]
 
                 # ghetto board copy to avoid exposing the real board to the engine
-                b = tilewe.Board(n_players=len(player_to_engine))
+                board_copy = tilewe.Board(n_players=len(player_to_engine))
                 for move in board.moves: 
-                    b.push(move) 
+                    board_copy.push(move) 
+                engine = self.engines[player_to_engine[board_copy.current_player]]
```

Comment:
with the discovery of the `current_player` as `bool` instead of `int` bug this change is irrelevant. but anyways, here it is getting current player from the replicated board instead of the master board, which is effectively a no-op/no change in functionality or performance